.env

```diff
@@ -0,0 +1 @@
+PASSWORD=
```

Comment:
@BestevenS You need to rebase with the latest changes that are merged in the main branch.